.github/workflows/docker-image-release.yml

```diff
@@ -0,0 +1,27 @@
+name: docker-image-release
+
+on: workflow_dispatch
```

Comment:
You can test this in this PR with
```
on:
    push:
```
And if it works, change it back to workflow dispatch